tests/test_chatpermissions.py

```diff
@@ -124,3 +124,13 @@ def test_equality(self):
 
         assert a != d
         assert hash(a) != hash(d)
+
+    def test_all_true(self):
+        f = ChatPermissions()
+        t = ChatPermissions.all_true()
+        # if the dirs are the same, the attributes will all be there
+        assert dir(f) == dir(t)
```

Comment:
maybe also add a `assert f != t`?

Reply:
sure, doesn't hurt.

Reply:
ah no, that gives me an assertionerror. So that == apparently doesn't care if the attributes Value is different.

Reply:
so in `all_true` we either need also add logic to dynamically update `self._id_attrs`, or just abandon dynamic approach and hardcode the `True` values and make it a `classmethod`

Reply:
okay so I think also dynamically updating the id sucks design wise. So I would go with hardcode and classmethod.